From fa200c2b59c0b8346466b503cd6ac80e331c9277 Mon Sep 17 00:00:00 2001
From: Fedora Ninjas <pjones@fedoraproject.org>
Date: Thu, 8 Mar 2012 13:43:39 -0800
Subject: [GRUB2] remove serial.mod test from 00_header (#748964)

This test doesn't belong here, upstream has removed it, and it
was blocking serial installs from working.
---
 util/grub.d/00_header.in |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/util/grub.d/00_header.in b/util/grub.d/00_header.in
index 5c4d9db..a1a75aa 100644
--- a/util/grub.d/00_header.in
+++ b/util/grub.d/00_header.in
@@ -113,10 +113,6 @@ for x in ${GRUB_TERMINAL_INPUT} ${GRUB_TERMINAL_OUTPUT}; do
 done
 
 if [ "x$serial" = x1 ]; then
-    if ! test -e "${GRUB_PREFIX}/serial.mod" ; then
-	echo "Serial terminal not available on this platform." >&2 ; exit 1
-    fi
-
     if [ "x${GRUB_SERIAL_COMMAND}" = "x" ] ; then
 	grub_warn "Requested serial terminal but GRUB_SERIAL_COMMAND is unspecified. Default parameters will be used."
 	GRUB_SERIAL_COMMAND=serial
-- 
1.7.7.6

